Source/WebCore/html/canvas/CanvasRenderingContext.h

```diff
@@ -91,6 +98,7 @@ class CanvasRenderingContext : public ScriptWrappable, public CanMakeWeakPtr<Can
 
     virtual bool compositingResultsNeedUpdating() const { return false; }
     virtual bool needsPreparationForDisplay() const { return false; }
+    // Swaps the current drawing buffer to display buffer.
     virtual void prepareForDisplay() { }
```

Comment:
This comment makes me think the function is incorrectly named.

Reply:
You're right. It might be prudent to change it to some sort of swapbuffers in future commits